packages/design-system/src/components/Icons/SvgIcon.tsx

```diff
@@ -74,7 +74,6 @@ export function SvgIcon({
     <svg
       aria-hidden={ariaHidden}
       className={svgClasses}
-      focusable={false}
```

Comment:
only needed for ie support - since we don't support ie anymore i removed it

Reply:
Nice!